examples/generative/corrdiff/conf/config_train_base.yaml

```diff
@@ -54,6 +54,8 @@ bench: false
   # Enable cuDNN benchmarking
 workers: 4
   # DataLoader worker processes
+songunet_checkpoint_level: 0
+  # Gradient checkpointing level, value is number of layers to checkpoint
```

Comment:
Can we clarify that 0 means no checkpointing?

Reply:
Updated with a note that indicates 0 is no checkpointing.